.rspec

```diff
@@ -1 +1,2 @@
 --require spec_helper
+--order rand
```

Comment:
What do you think about uncommenting https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/f1a1db52e204d2be2e8edb5c0b403404ecf03c34/spec/spec_helper.rb#L88, so that all the spec config is in the same place?

Reply:
Yes, this is much better